### ISSUE TITLE
Slider bar on jobs panel cuts off status on right and looks a little funky

### ISSUE THREAD

**Issue by mattdawkins** [Kitware/dive#1164]
**Describe the bug**
The slider bar at the bottom of the jobs panel is very skinny (some people might not know it's a slider) and it's presence leads to cutting off the job status items on the right sometimes. I think it use to never have a slider but could be mis-remembering. 

![bar](https://user-images.githubusercontent.com/2353972/153667146-e533992a-83cc-4c9f-b1e3-7a9998b298b3.png)

**Expected behavior**
Maybe the text could be cut off in the first column as opposed to even having a slider? Depending on browser width the user might not even know the job status is present when it's cut off at the perfect place (happened to me a couple times)

**Comment by mattdawkins** [comment]
Pretty minor issue FWIW

**Comment by waxlamp** [comment]
I agree that the scroll bar is hard to see and use when there's horizontal overflow. Thanks for reporting it.
